docs/implementation.html

```diff
@@ -5,9 +5,9 @@
  The ASF licenses this file to You under the Apache License, Version 2.0
  (the "License"); you may not use this file except in compliance with
  the License.  You may obtain a copy of the License at
- 
+
```

Comment:
No it was just line space. Sorry I didn't catch that, I auto cleanup whitespace on files I open/modify and I must have touched this file. Should I re-pull or do you want to keep it?